<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PictureProcessForm</class>
 <widget class="QWidget" name="PictureProcessForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>986</width>
    <height>847</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="displayArea" native="true"/>
   </item>
   <item>
    <widget class="QWidget" name="operratingFloor" native="true">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QSlider::groove:horizontal {
border: -4px solid;
height: 14px;
left: 3px; right: 3px;
}
QSlider::handle:horizontal {
background: qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 #fdfcfb, stop:1 #e2d1c3);
border: 1px solid #5c5c5c;
width: 14px;
margin: -1 0 -1 1;
border-radius: 8px;
}
QSlider::add-page:horizontal{
background: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:1 #BDBDBD, stop:0 #DBDBDB);

}
QSlider::sub-page:horizontal{
background: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 #596164, stop:1 #868f96);
}
QRadioButton{
    margin: 0 0 0 6
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="hslBtn">
        <property name="text">
         <string>色相/饱和度</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bwBtn">
        <property name="text">
         <string>黑白调整</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="lcBtn">
        <property name="text">
         <string>亮度/对比度</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="scBtn">
        <property name="text">
         <string>可选颜色</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="resetBtn">
        <property name="text">
         <string>复位</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>750</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="closeBtn">
        <property name="text">
         <string>关闭</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveBtn">
        <property name="text">
         <string>保存</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
